devices/Button/README.md

```diff
@@ -0,0 +1,87 @@
+﻿# Button
+
+The ButtonBase class is a base implementation for buttons that is hardware independent and can be used across devices.
+The GpioButton is a GPIO implementation of the button and inherits from the ButtonBase class. This implementation has been tested on an ESP32 platform, specificaly on the [M5StickC Plus](https://shop.m5stack.com/products/m5stickc-plus-esp32-pico-mini-iot-development-kit).
+
+## Documentation
+
+Documentation for the M5StickC Plus, including pin mapping, can be [found here](https://docs.m5stack.com/en/core/m5stickc_plus).
```

Comment:
rather than the documentation on the M5Stick, maybe better to point on a button event sequence? Or on top of this?

Reply:
Thanks, should be added now.